policyengine_us/parameters/gov/states/ri/tax/income/exemption/amount.yaml

```
@@ -0,0 +1,19 @@
+description: Rhode Island provides the following amount for each exemption in the tax unit.
+values:
+  2021-01-01: 4_250
+  2022-01-01: 4_350
+
+
+metadata:
+  label: Rhode Island exemption amount 
+  period: year
+  unit: currency-USD
+  reference:
+  # The tax form points to this worksheet: Exemption Worksheet for RI-1040NR, Page 1, line 6)
+  - title: Rhode Island Tax Rate Schedule and Worksheet 2022 (Exemption Worksheet line 1)
+    href: https://tax.ri.gov/sites/g/files/xkgbur541/files/2022-12/2022%20Tax%20Rate%20and%20Worksheets.pdf
```

in all pdfs (even if only one page)
```suggestion
    href: https://tax.ri.gov/sites/g/files/xkgbur541/files/2022-12/2022%20Tax%20Rate%20and%20Worksheets.pdf#page=1
```